package.json

```diff
@@ -11,16 +11,14 @@
     "addons/storyshots/*",
     "app/*",
     "lib/*",
-    "examples/marko-cli",
-    "examples/angular-cli",
+    "examples/*-cli",
     "examples/cra-kitchen-sink",
     "examples/html-kitchen-sink",
```

Comment:
Could have done `examples/*-kitchen-sink` too if not for `crna-kitchen-sink` package.